packages/clay-button/src/ClayButton.js

```diff
@@ -39,13 +39,22 @@ ClayButton.STATE = {
 	/**
 	 * The button disabled attribute.
 	 * @instance
-	 * @memberof ButtonBase
+	 * @memberof ClayButton
 	 * @type {?boolean}
 	 * @default false
 	 */
 	disabled: Config.bool().value(false),
 
 	/**
+	 * CSS classes to be applied to the element.
+	 * @instance
+	 * @memberof ClayButton
+	 * @type {?string|undefined}
+	 * @default false
+	 */
+	elementClasses: Config.string(),
```

Comment:
Just as a note, technically these components will already inherit the `elementClasses` prop from https://github.com/metal/metal.js/blob/master/packages/metal-component/src/Component.js#L664-L672, but I actually like the visibility of having it here. API docs generation in the future will also be easier this way.

Reply:
Hey @robert-frampton yep, we did It in this way in FARO because of that 😃